doc/ref_cert/lfn/chapters/chapter09.md

```diff
@@ -62,3 +65,75 @@ test cases must pass as they are for CNTT NFVI Conformance/Certification:
 | heat_ims                   | opensource VNF onboarding and testing                                    |
 | vyos_vrouter               | opensource VNF onboarding and testing                                    |
 | juju_epc                   | opensource VNF onboarding and testing                                    |
+
+<a name="9.4"></a>
+## 9.4 CNTT Compliance Cookbook
+
+[Xtesting CI](https://galaxy.ansible.com/collivier/xtesting) only requires
+GNU/Linux as Operating System and asks for a few dependencies as described in
+[Deploy your own Xtesting CI/CD toolchains](https://wiki.opnfv.org/pages/viewpage.action?pageId=32015004):
+- python-virtualenv
+- docker.io
+- git
+
+Please note the next two points depending on the GNU/Linux distributions and
+the network settings:
+- SELinux: you may have to add -\-system-site-packages when creating the
+  virtualenv ("Aborting, target uses selinux but python bindings
+  (libselinux-python) aren't installed!")
+- Proxy: you may set your proxy in env for Ansible and in systemd for Docker
+  https://docs.docker.com/config/daemon/systemd/#httphttps-proxy
+
+At the time of writing, the CI description file is hosted in Functest and only
+runs the containers listed in RM/RA-1 Requirements. It will be completed by the
+next CNTT mandatory test cases and then a new CI description file will be
+proposed in CIRV tree.
+
+To deploy your own CI toolchain running CNTT Compliance:
+```bash
+virtualenv functest
+. functest/bin/activate
+pip install ansible
+ansible-galaxy install collivier.xtesting
+git clone https://gerrit.opnfv.org/gerrit/functest functest-src
+(cd functest-src && git checkout -b stable/hunter origin/stable/hunter)
+ansible-playbook functest-src/ansible/site.cntt.yml
+```
+
+<a name="9.4.1"></a>
+### 9.4.1 NVFI API testing configuration
```

Comment:
NVFI -> NFVI

Comments:
- I'd add a note that this cookbook assumes the reader is already familiar with the tools and systems involved. In other words, this is not a How-To, nor is it step-by-step. This is guidance for experience users to tailor for CNTT RC.
- We should run through this in the lab, but not before Prague. Cedric has provided the framework, but since he is so familiar with the process, gaps may not be as apparent to him, as for the rest of us. I'd recommend we approve this PR, and create an Issue to go take it for a drive.

Reply:
Thank you for the typo. fixing.

Here it leverages on copy/paste operations whatever the skills.
Yes it can make sens to duplicate Functest doc here (e.g. getting openstack creds via horizon) if needed. Thank you for the feedback. I fully agree to merge it as a first step.